hooks/environment

```diff
@@ -24,6 +24,18 @@ load_secret_into_env() {
   export "${export_name}=${secret_value}"
 }
 
+load_all_secrets_into_env() {
+  local secret_id="$1"
+  local json_key="$2"
+  local secret_value
+  echo "Reading all environment variables from ${secret_id} in AWS SM"
+  secret_value="$(get_secret_value "${secret_id}")"
+  for s in $(echo ${secret_value} | jq "${json_key} | to_entries[]" | jq -r '.key + "=" + .value'); do
```

Comment:
I think this loop will break on spaces within secret keys or values, worth at least adding a test to this effect.

E.g.

```
$ export json_key='.'
$ export secret_value='{"a":"v1", "b": "v2 v3", "c d": "v4"}'
$ for s in $(echo ${secret_value} | jq "${json_key} | to_entries[]" | jq -r '.key + "=" + .value'); do
for> echo "$s"
for> done
a=v1
b=v2
v3
c
d=v4
```